apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -153,6 +153,9 @@ type WorkspaceConfig struct {
 	// if the workspace's Template Spec Components are not defined. The DefaultTemplate will overwrite the existing
 	// Template Spec, with the exception of Projects (if any are defined).
 	DefaultTemplate *dw.DevWorkspaceTemplateSpecContent `json:"defaultTemplate,omitempty"`
+	// DefaultSchedulerName is the name of the pod scheduler for DevWorkspace pods.
+	// If not specified, the pod scheduler is set to the default scheduler.
+	DefaultSchedulerName string `json:"defaultSchedulerName,omitempty"`
```

Comment:
Slightly unclear -- field is `DefaultSchedulerName` and value if not set is the default scheduler.
```suggestion
	// DefaultSchedulerName is the name of the pod scheduler for DevWorkspace pods.
	// If not specified, the pod scheduler is set to the default scheduler on the cluster.
	DefaultSchedulerName string `json:"defaultSchedulerName,omitempty"`
```
We should also consider renaming field to just `schedulerName` to match some other fields (`ImagePullPolicy`, `StorageClassName`, etc.)

Reply:
Sounds good, changed to `schedulerName` for now cc @ibuziuk